SECURITY_INFORMATION_MANAGEMENT_POLICY.md

```diff
@@ -0,0 +1,18 @@
+# SECURITY INFORMATION MANAGEMENT POLICY
+
+## Why do we need your username and password for different accounts?
+We need your username and password to access third party applications. Since we are not in a web browser, we do not have the capability of accessing your information through redirect pages. 
+In order to complete the items below, we need to authenticate into your account, which requires us to use Basic Authentication which needs a username and password. 
+If you feel uncomfortable sharing your login credentials, you can simply skip using any third party tools. If you'd like to use third party tools, but don't want to share your credentials, 
+check out our Web Application, [Yggdrasil](https://github.com/hammer-io/yggdrasil), which is coming soon. 
+
+In the mean time, we urge you to check out the source code and see exactly how we are using your username and password. 
+
+## What do we do with your username and password?
+* Read your GitHub account details, create a GitHub token, delete a GitHub token, create a GitHub repository, push to your GitHub repository through the official (GitHub API)[https://developer.github.com/v3/].
```

Comment:
These two bullets are a bit vague. I think they could be clarified as follows:
- If you opt to use GitHub for source control, your credentials are used to create the new project repository, push the generated source code to the new repository, and facilitate other actions relevant to the Tyr project.
- If you opt to use TravisCI for continuous integration, your GitHub credentials are used to activate and configure the Tyr project in the TravisCI environment.
- If you opt to deploy your project on Heroku, your credentials are used to create and configure the Heroku Application for the Tyr project.
- In general, as more third party applications are integrated into Tyr, the credentials we request are simply used to facilitate DevOps activities for your project.